enhancements/sig-architecture/6-placements/README.md

```diff
@@ -0,0 +1,523 @@
+10. Otherwise, return clusters in the intersection with some random additional clusters from the remaining of the new matched clusters;
+
+**Note:** The old matched clusters are preferred during the predicate evaluation, which makes the placement decisions as stable as possible.
+
+### Open Questions
+
+1. The API group of the new placement APIs could be `cluster.open-cluster-management.io` or `scheduling.open-cluster-management.io`. Which one is better?
+
+### Test Plan
+
+- Unit tests will cover the functionality of the controllers.
+- Unit tests will cover the new APIs.
+- Integration tests will cover all user stories.
+- And e2e tests will cover the following cases.
+  - Create/update/delete a placement;
+  - ManagedCluster labels/claims changes;
+  - A ManagedClusterSet is bound/unbound to/from the placement namespace;
+  - A ManagedCluster is added/removed to/from a bound ManagedClusterSet;
+
+### Graduation Criteria
+
+#### Alpha
+1. The new APIs is reviewed and accepted;
+2. Implementation is completed to support the functionalities;
+3. Develop test cases to demonstrate that the above user stories work correctly;
+
+#### Beta
+1. The new APIs is able to cover all the capabilities of the current placementrule APIs;
+2. Conversion webhook is provided to migrate existing placement objects to the new placement APIs;
+3. At lease one component uses the new APIs;
+4. Support selecting clusters belong to no ManagedClusterSet;
+5. Support tolerations/taints (user story 8);
+6. Create metrics data when creating/updating placement decisions (user story 9);
+7. Support time window (user story 10);
+
+#### GA
+1. Pass the performance/scalability testing;
+2. Migrate all components which has dependency on the current placementrule APIs to the new placement APIs;
+3. Deprecate the current placementrule APIs, and it will still exist in the next 3 releases.
+
+### Upgrade Strategy
+
+#### Placement Manager
+Placement Manager is a new component and contains controllers to implement the new placement APIs. 
+1. A new repo is required for `placement-manager`;
+2. Change on `ClusterManager` CRD is required to include information for `placement-manager`;
+3. Change on `cluster-manager` is required to
+    - Install the new placement CRD on hub cluster;
+    - Deploy `placement-manager`;
+
+#### API Migration
+For those components who use the current placementrule API and want to migrate to the new placement APIs, the strategy below should be followed:
+1. The new placement APIs will co-exists with the current placementrule API in at least 3 releases;
+2. The objects of the current placementrule API will be converted into objects of the new placement APIs with the same name by conversion webhook;
+3. Mannual operation is required to make the converted new placement objects work by binding necessary clustersets to the placement namespace.
```

Comment:
This bullet is also sounding to me like conversion into the new APIs will need to be an offline process

Reply:
Yes, you are right. It should be an offline process. And a conversion tool will be provided instead.